sympy/sets/fancysets.py

```diff
@@ -1433,6 +1431,11 @@ def _intersect(self, other):
 
     def _union(self, other):
 
+        if other.is_subset(S.Reals):
+
```

Comment:
PEP-8 says: Use blank lines in functions, sparingly, to indicate logical sections.